database/orm/model.go

```diff
@@ -31,6 +31,6 @@ type SoftDeletes struct {
 }
 
 type Timestamps struct {
-	CreatedAt time.Time
-	UpdatedAt time.Time
+	CreatedAt carbon.DateTime `gorm:"autoCreateTime;column:created_at"`
+	UpdatedAt carbon.DateTime `gorm:"autoUpdateTime;column:updated_at"`
```

Comment:
> Q: Will this change affect the existing logic?

It has little effect, but it will be automatically parsed into Datetime format when Gin returns, likes Laravel.